translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+      }
+    ],
+    "responses": {
+      "title": ["{name} Felhasználói Információja"],
+      "userInfoName": ["Felhasználó Információ"],
+      "userInfoValue": [
+        "Név: {name}\n",
+        "Diszkriminátor: {discriminator}\n",
+        "ID: {id}\n",
+        "Készítve: <t:{created}:F>"
+      ],
+      "memberInfoName": ["Tag információ"],
+      "boosted": ["Szervergyorsító <t:{time}:F> óta"],
+      "memberInfoValue": [
+        "Becenév: {nick}\n",
+        "Legmagasabb role: {top_role}\n",
+        "Csatlakozott: <t:{joined}:F>\n",
+        "{boosted}"
+      ],
+      "playerInfoName": ["Player Information"],
+      "playerInfoValue": [
+        "Felhasználónév: {name}\n",
+        "Rang: {rank}\n",
+        "Hypixel Szint: {level}\n",
+        "Guild: {guild}"
+      ]
+    }
+  },
+  "verify": {
+    "name": "ellenőrzés",
+    "description": "Ellenőrző parancsok."
+  },
+  "verify link": {
+    "name": "csatlakoztatás",
+    "description": "Csatlakoztasd a fiókod a Hypixel social menüjén keresztül.",
+    "options": [
+      {
+        "name": "játékos",
+        "description": "A játékos akit ellenőrizni szeretnél."
+      }
+    ],
+    "responses": {
+      "notLinked": [
+        "**{name}**-nek nincsen Discord fiókja összekötve a Hypixel social menün keresztül.\n\n",
+        "Hogy megtanuld, hogyan kösd össze a Discord fiókod a Hypixel social menün keresztül, próbáld ki a `/verify help` parancsot."
+      ],
+      "notVerified": [
+        "**{name}**-nek az összekötött Discord fiókja nem azonos a tiéddel.\n\n",
+        "Hogy megtanuld, hogyan kösd össze a Discord fiókod a Hypixel social menün keresztül, próbáld ki a `/verify help` parancsot."
+      ],
+      "alreadyVerified": [
+        "**{name}** már ellenőrizve van a <@{userid}> ID-vel. Az ellenőrzés el fogja távolítani ezt az összekötést a fiókon.\n\n",
+        "Szeretnéd folytatni?"
+      ],
+      "cancelled": [
+        "Ellenőrzés megszakítva."
+      ],
+      "verified": [
+        "**{name}** össze lett kötve a Discord fiókoddal."
+      ]
+    }
+  },
+  "verify help": {
+    "name": "segítség",
+    "description": "Tanuld meg, hogy hogyan kösd össze a Discord fiókodat a Hypixel social menün keresztül.",
+    "options": [],
+    "responses": {
+      "title": ["Ellenőrző Instrukciók"],
+      "description": [
+        "Hogy hozzákösd a Minecraft fiókodat a Discord fiókodhoz:\n",
+        "**1)** Akármelyik Hypixel lobbyban jobb-klikkelj a \"My Profile\" tárgyadra a hotbar második helyén.\n",
+        "**2)** Válaszd ki a \"Social Media\" opciót, jobbra a Redstone Blokktól.\n",
+        "**3)** Válaszd ki a \"Discord\" opciót.\n",
+        "**4)** Másold ki a Discord felhasználónevedet ({name}) a chatbe. __Ez kis- és nagybetűérzékeny.__\n",
+        "**5)** Ha egy könyv felbukkan, válaszd a \"I understand\" opciót.\n",
+        "**6)** Használd a `/verify link` parancsot, hogy hozzákösd a Minecraft fiókod a Discord fiókodhoz.\n",
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`